tools/PI/DevHome.PI/Views/BarWindowVertical.xaml.cs

```diff
@@ -59,9 +66,25 @@ private void MainPanel_Loaded(object sender, RoutedEventArgs e)
         RemoveThickFrameAttribute();
 
         // Regardless of what is set in the XAML, our initial window width is too big. Setting this to 70 (same as the XAML file)
-        Width = 70;
-    }
-
+        Width = 70;
+
+        // Calculate the DPI scale.
+        var monitor = PInvoke.MonitorFromWindow(ThisHwnd, MONITOR_FROM_FLAGS.MONITOR_DEFAULTTONEAREST);
+        PInvoke.GetScaleFactorForMonitor(monitor, out DEVICE_SCALE_FACTOR scaleFactor).ThrowOnFailure();
+        _dpiScale = (double)scaleFactor / 100;

```

Comment:
I think the Horizontal bar has this problem too... if I move the bar to a monitor with a different DPI, this will break, right? Almost like we need to update the DPI on move...

Reply:
yeah this needs more work to be DPI correct, I'll file a task because I think we can do this more holistically somehow but I don't want to complicate this PR anymore.